applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -351,6 +351,7 @@ void KratosGeoMechanicsApplication::Register()
     KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElasticPlaneStrainK02DLaw", mLinearPlaneStrainK0Law)
     KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElasticK03DLaw", mElasticIsotropicK03DLaw)
     KRATOS_REGISTER_CONSTITUTIVE_LAW("GeoLinearElasticPlaneStrain2DLaw", mLinearElasticPlaneStrain2DLaw)
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("GeoLinearElastic3DLaw", mLinearElastic3DLaw)
```

Comment:
I think we should have the 'incremental' specifier in the registered name (maybe GeoIncrementalLinearElastic3DLaw?), but it's good to also have @WPK4FEM's opinion on this)

Reply:
I share Richards point of view.